samples/kubernetes/end2end/docs/05-prometheus.md

```diff
@@ -1,19 +1,20 @@
-## Installing Prometheus
-Install Prometheus using a [Helm chart](https://github.com/helm/charts/tree/master/stable/prometheus) with the customized [values.yaml](../prometheus/values.yaml) file.
+## Setting up Prometheus
+Install Prometheus using a [Helm chart](https://github.com/helm/charts/tree/master/stable/prometheus) with the customized [values.yaml](../prometheus/values.yaml). There are four components will be installed with this chart: Prometheus server, Alertmanager, node-exporter and kube-state-metrics.  
```

Comment:
Replace "There are four components will be installed with this chart:" with "There are four components that will be installed with this chart:"